components/delivery-postcode.jsx

```diff
@@ -7,6 +7,16 @@ const postcodeLabel = {
 	CAN: 'Postal Code',
 };
 
+const promptLabel = {
+	USA: 'Please enter your five digit Zip Code',
```

Comment:
For consistency with CAN and to match the requirements
```suggestion
	USA: 'Please enter your 5 digit Zip Code',
```

Reply:
done!